### PR TITLE
[Service] error code when parsing json

### PR DESCRIPTION
**Pull request by jaeyun-jung**
Change error code when parsing json string of config file.

### CODE REVIEW

**Review by niley7464 — APPROVED** [review]
LGTM!

**Review by songgot — APPROVED** [review]
LGTM

File: c/src/ml-api-service.c [hunk 1]
```diff
@@ -339,8 +339,8 @@ ml_service_new (const char *config, ml_service_h * handle)
   }
 
   if (!json_parser_load_from_data (parser, json_string, -1, &err)) {
-    _ml_error_report_return (ML_ERROR_INVALID_PARAMETER,
-        "Failed to parse configuration file. Parse error: %s",
+    _ml_error_report_return (ML_ERROR_IO_ERROR,
```
Comment (anyj0527) [inline c/src/ml-api-service.c:342]: How about keep old lines? I think Json parse error means it succeed to load the json but fail to parse it. The existence of json file is checked already.

Comment (jaeyun-jung) [reply c/src/ml-api-service.c:342]: agreed, close this pr.

**Pull request closed without merging**